platform/platform-scheduler/src/main/java/ddf/platform/scheduler/CommandJob.java

```diff
@@ -44,7 +47,20 @@
     private CommandProcessor commandProcessor;
 
     @Override
-    public void execute(JobExecutionContext context) throws JobExecutionException {
+    public void execute(final JobExecutionContext context) throws JobExecutionException {
+        Subject subject = getDdfSubjectUtils().getSystemSubject();
+        if (subject != null) {
+            subject.execute(new Callable() {
+                @Override
+                public Object call() throws Exception {
+                    doExecute(context);
+                    return null;
+                }
+            });
+        }
```

Comment:
Should we log a warning if the command was not able to get a subject and execute?